### PR TITLE
Create pinkyplant.json

### PR DESCRIPTION
**Pull request by pinkyplant**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

pinkyplant.github.io

### CODE REVIEW

**Comment by orxngc** [comment]
image
Inaccessible site as well.

**Comment by pinkyplant** [comment]
Hi, @orxngc
The website should now be accessible.
Thanks.
Jay.

File: domains/pinkyplant.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Jay's website and portfolio.",
+    "repo": "https://github.com/pinkyplant/pinkyplant.github.io",
+    "owner": {
+        "username": "pinkyplant",
+        "email": "pinkyplantz@proton.me",
+        "twitter": "pinkyplant"
+    },
+    "record": {
+        "CNAME": "pinkyplant.github.io"
```
Comment (CuteDog5695) [inline domains/pinkyplant.json:10]: Please add more stuff to your website (for example an about me)

**Review by iostpa — DISMISSED** [review]

**Comment by orxngc** [comment]
You fixed the inaccessibility issue but its still incomplete,
; I need to see some sort of content on the site.

**Comment by pinkyplant** [comment]
That is the content @orxngc

File: domains/pinkyplant.json (hunk 1 above)
Comment (pinkyplant) [reply domains/pinkyplant.json:10]: Okay thank you. Will resolve ASAP

**Pull request merged**